test/e2e/storage/testsuites/volumemode.go

```diff
@@ -482,16 +482,6 @@ func listPodDirectory(h utils.HostExec, path string, node *v1.Node) ([]string, e
 	return strings.Split(out, "\n"), nil
 }
 
-// NOTE(avalluri): The below code is intentionally copied from e2e/common package.
-// Testsuites depending on common package is not desirable as that pulls quite
-// many tests which are not interested by storage suites.
-//
-// waitTimeoutForEvent waits the given timeout duration for an event to occur.
-func waitTimeoutForEvent(c clientset.Interface, namespace, eventSelector, msg string, timeout time.Duration) error {
-	interval := 2 * time.Second
-	return wait.PollImmediate(interval, timeout, eventOccurred(c, namespace, eventSelector, msg))
```

Comment:
I think the `eventOccurred` below can now also be removed.

Reply:
Yep, I miss it. Thanks.
PR updated, PTAL